trivia.py

```diff
@@ -359,6 +365,13 @@ def _stop(self, *args):
             self._save_game()
             self.factory.running = False
 
+    def _write_log(self, logfile, f, q):
+        '''
+        Writes broken and reported questions to log file
+        '''
+        with open(config.SAVE_DIR+logfile, 'a') as log:
+            log.write(f+'\n'+q+'\n')
```

Comment:
Probably put the filename, line number and question/answer all on the same line in the log file. I'm not too worried about it wrapping, since anyone debugging will be mostly interested in the file and line number.